chore: prepare next.clayui.com for release v3.0.0-milestone.1 (#2101)
liferay/clay: next.clayui.com/content/docs/migration/migrate-the-clay-components-from-v2-to-v3.md

Clay v3 is introducing the rewriting of components in React and new components, the Framework change leads to some API changes that may no longer make sense for a context with React, wait for API changes and renames.

> We are also betting on the delivery of components oriented to composition, simple components that do very well what is defined, this increases the space for flexibility and more customization but also increase the work of composing but we will be delivering the components with a higher level of logic also, you can notice the nomenclature:

Reviewer: ```suggestion
These components are intended to be consumed via composition, this increases flexibility and more customization. We will also be delivering components with a higher level of logic as well, you can notice the nomenclature:
```

> - `<ClayDropDown.Item />`: does render an item very well.

Reviewer: ```suggestion
- `<ClayDropDown.Item />`: renders basic markup.
```

> - `<ClayDropDownWithPagination />`: renders a number of items and some more logic.

Reviewer: ```suggestion
- `<ClayDropDownWithPagination />`: renders a number of items and includes logic for functionality.
```
